app/views/shared/_alternate_page_links.html.erb

```diff
@@ -0,0 +1,3 @@
+<% I18n.available_locales.each do |locale| %>
```

Comment:
Should this skip over the current locale?

Reply:
No, [the examele here](https://developers.google.com/search/docs/advanced/crawling/localized-versions#example) shows the current locale should be present with a `hreflang="x-default"`.